Javascript/Goodboy-Oldboy /script.js

```diff
@@ -0,0 +1,18 @@
+let dogYearBirth = 2014;
+let dogYearFuture = 2031;
+let shouldShowResultInDogYears = true;
+
+let dogAge;
+if (shouldShowResultInDogYears) {
```

Comment:
You repeat this line of code twice. Why not combine the logic at one place?

Reply:
***Mentor, you mean to use the ternary operator?****

Reply:
![Screenshot 2024-06-05 at 5 19 05 PM](https://github.com/l0rians/hyf-homework/assets/52400967/64cc8f58-fd22-4afe-bc13-40b141b8ec9f)
The highlighted is repeated. You could combine it into a single `if else`.